assets/js/state/hosts.js

```diff
@@ -79,6 +79,22 @@ export const hostsListSlice = createSlice({
         return host;
       });
     },
+    setHostDeregistering: (state, action) => {
+      state.hosts = state.hosts.map((host) => {
+        if (host.id === action.payload.id) {
+          return { ...host, deregistering: true };
+        }
+        return host;
+      });
+    },
+    setHostNotDeregistering: (state, action) => {
```

Comment:
I have a doubt about naming. I would have thought of something that talks about _starting_ a deregistration and _completing_ a deregistration.
No hard feelings tho :smile: